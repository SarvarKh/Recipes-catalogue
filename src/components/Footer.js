
export default function Footer() {
    return (
      <>
        <footer>
            <div>
                Copyright Â© 2021 Sarvar Khalimov
                <a className="icon-link" href="https://github.com/SarvarKh" target="_blank"><i class="fab fa-github" aria-hidden="true"></i></a>
                <a className="icon-link" href="https://www.linkedin.com/in/sarvar-khalimov/" target="_blank"><i class="fab fa-linkedin" aria-hidden="true"></i></a>
                <a className="icon-link" href="https://sarvarkh.github.io/portfolio/" target="_blank"><i class="fas fa-folder-open" aria-hidden="true"></i></a>
            </div>
        </footer>
      </>
    );
  }
  