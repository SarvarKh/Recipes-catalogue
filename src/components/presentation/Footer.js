export default function Footer() {
  return (
    <>
      <footer>
        <div>
          Copyright Â© 2021 Sarvar Khalimov
          <a className="icon-link" href="https://github.com/SarvarKh" target="_blank" rel="noreferrer">
            <i className="fab fa-github" aria-hidden="true" />
          </a>
          <a className="icon-link" href="https://www.linkedin.com/in/sarvar-khalimov/" target="_blank" rel="noreferrer">
            <i className="fab fa-linkedin" aria-hidden="true" />
          </a>
          <a className="icon-link" href="https://sarvarkh.github.io/portfolio/" target="_blank" rel="noreferrer">
            <i className="fas fa-folder-open" aria-hidden="true" />
          </a>
        </div>
      </footer>
    </>
  );
}
