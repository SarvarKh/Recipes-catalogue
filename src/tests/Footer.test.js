import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/presentation/Footer';

it('renders correctly', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
