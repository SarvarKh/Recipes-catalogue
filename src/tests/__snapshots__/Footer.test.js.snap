// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer>
  <div>
    Copyright Â© 2021 Sarvar Khalimov
    <a
      className="icon-link"
      href="https://github.com/SarvarKh"
      rel="noreferrer"
      target="_blank"
    >
      <i
        aria-hidden="true"
        className="fab fa-github"
      />
    </a>
    <a
      className="icon-link"
      href="https://www.linkedin.com/in/sarvar-khalimov/"
      rel="noreferrer"
      target="_blank"
    >
      <i
        aria-hidden="true"
        className="fab fa-linkedin"
      />
    </a>
    <a
      className="icon-link"
      href="https://sarvarkh.github.io/portfolio/"
      rel="noreferrer"
      target="_blank"
    >
      <i
        aria-hidden="true"
        className="fas fa-folder-open"
      />
    </a>
  </div>
</footer>
`;
